projects/admin/src/app/classes/loans.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2021 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/* tslint:disable */
+// required as json properties is not lowerCamelCase
+
+import { marker } from '@biesbjerg/ngx-translate-extract-marker';
```

Comment:
in other PR, I see `import { marker as _ } from '@biesbjerg/ngx-translate-extract-marker';` what is better?

Reply:
I tried but it seems to have a problem too use this syntax into `enum`. This syntax is already used in `items.ts` file